### DISCUSSION TITLE
Beeper component with pitch

### DISCUSSION THREAD

**Discussion by FalconPilot** [FakeFishGames/Barotrauma#13679] [Ideas and suggestions]
Hello! I really enjoy sub building and mostly thinking about "how a sub's crew can engage with the way the sub works". And one thing I really thought one day was that it would be really awesome to have a submarine module with a sound-based recognition pattern (kinda like a motherboard BIOS signal or an aircraft RWR). A simple component that emits a "BEEP" noise with an input to set the pitch of the beeping sound would be a simple yet very cool way to implement such a concept 😄

**Comment by BlackMoons** [comment]
100% we need more alarm noises in general. And pitch bending should be trivial for most sound API's (Its just a parameter you feed when you play the sound)

**Comment by Cherman0** [comment]
If this gets added it will take about 3 seconds for someone to make baro rickroll sub. I approve
